Migration fails where there is no previous address.json

Steps to reproduce:

1. On your localhost, delete ${HOME}/.ocean/ocean-contracts/artifacts/address.json
2. start barge:   ./start_ocean.sh --no-dashboard --no-provider --no-aquarius

Ocean-contracts will fail with:

cean-contracts_1_961411a02a92 |    > Saving migration to chain.
ocean-contracts_1_961411a02a92 |    > Saving artifacts
ocean-contracts_1_961411a02a92 |    -------------------------------------
ocean-contracts_1_961411a02a92 |    > Total cost:          0.00287858 ETH
ocean-contracts_1_961411a02a92 | 
ganache_1_c5b436a249d3 | eth_getBlockByNumber
ocean-contracts_1_961411a02a92 | 
ocean-contracts_1_961411a02a92 | 2_deploy_contracts.js
ocean-contracts_1_961411a02a92 | =====================
ganache_1_c5b436a249d3 | eth_accounts
ocean-contracts_1_961411a02a92 | 
**_ocean-contracts_1_961411a02a92 | Error: ENOENT: no such file or directory, open './artifacts/address.json'_**
ocean-contracts_1_961411a02a92 |     at Object.openSync (fs.js:462:3)
ocean-contracts_1_961411a02a92 |     at Object.readFileSync (fs.js:364:35)
ocean-contracts_1_961411a02a92 |     at /ocean-contracts/migrations/2_deploy_contracts.js:17:44
ocean-contracts_1_961411a02a92 |     at /ocean-contracts/node_modules/truffle/build/webpack:/packages/deployer/index.js:64:1
ocean-contracts_1_961411a02a92 |     at /ocean-contracts/node_modules/truffle/build/webpack:/packages/deployer/src/deferredchain.js:20:1
ocean-contracts_1_961411a02a92 |     at processTicksAndRejections (internal/process/task_queues.js:97:5)
ocean-contracts_1_961411a02a92 |     at Migration._deploy (/ocean-contracts/node_modules/truffle/build/webpack:/packages/migrate/Migration.js:68:1)
ocean-contracts_1_961411a02a92 |     at Migration._load (/ocean-contracts/node_modules/truffle/build/webpack:/packages/migrate/Migration.js:55:1)
ocean-contracts_1_961411a02a92 |     at Migration.run (/ocean-contracts/node_modules/truffle/build/webpack:/packages/migrate/Migration.js:171:1)
ocean-contracts_1_961411a02a92 |     at Object.runMigrations (/ocean-contracts/node_modules/truffle/build/webpack:/packages/migrate/index.js:150:1)
ocean-contracts_1_961411a02a92 |     at Object.runFrom (/ocean-contracts/node_modules/truffle/build/webpack:/packages/migrate/index.js:110:1)
ocean-contracts_1_961411a02a92 |     at Object.run (/ocean-contracts/node_modules/truffle/build/webpack:/packages/migrate/index.js:87:1)
ocean-contracts_1_961411a02a92 |     at runMigrations (/ocean-contracts/node_modules/truffle/build/webpack:/packages/core/lib/commands/migrate.js:269:1)
ocean-contracts_1_961411a02a92 |     at /ocean-contracts/node_modules/truffle/build/webpack:/packages/core/lib/commands/migrate.js:231:1
ocean-contracts_1_961411a02a92 | Truffle v5.1.43 (core: 5.1.43)
ocean-contracts_1_961411a02a92 | Node v12.19.0
ocean-contracts_1_961411a02a92 | npm ERR! code ELIFECYCLE
ocean-contracts_1_961411a02a92 | npm ERR! errno 1
ocean-contracts_1_961411a02a92 | npm ERR! @oceanprotocol/contracts@0.5.5 deploy:development: `npm run clean && npm run compile && export NETWORK='development' && truffle migrate && npm run export:artifacts`
ocean-contracts_1_961411a02a92 | npm ERR! Exit status 1
ocean-contracts_1_961411a02a92 | npm ERR! 
ocean-contracts_1_961411a02a92 | npm ERR! Failed at the @oceanprotocol/contracts@0.5.5 deploy:development script.
ocean-contracts_1_961411a02a92 | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
ocean-contracts_1_961411a02a92 | 
ocean-contracts_1_961411a02a92 | npm ERR! A complete log of this run can be found in:
ocean-contracts_1_961411a02a92 | npm ERR!     /root/.npm/_logs/2020-10-12T16_33_20_798Z-debug.log

--- a/migrations/2_deploy_contracts.js
+++ b/migrations/2_deploy_contracts.js
@@ -15,9 +15,9 @@ module.exports = function(deployer, network, accounts) {
     deployer.then(async () => {
         const addressFile = './artifacts/address.json'
         let oldAddresses
-        try{
+        try {
             oldAddresses = JSON.parse(fs.readFileSync(addressFile))
-        }catch(e){  oldAddresses={} }
+        } catch (e) { oldAddresses = {} }
         const networkName = process.env.NETWORK
         if (!oldAddresses[networkName]) {
             oldAddresses[networkName] = {}

